<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1106</width>
    <height>649</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normalon>:/yang.png</normalon>
   </iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <widget class="QWidget" name="page"/>
      <widget class="QWidget" name="page_2"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1106</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuYonghu">
    <property name="title">
     <string>用户(&amp;U)</string>
    </property>
    <addaction name="actionLogin"/>
    <addaction name="actionLogout"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_I">
    <property name="title">
     <string>信息管理(&amp;I)</string>
    </property>
    <addaction name="actionD"/>
    <addaction name="action_Device_Query"/>
   </widget>
   <widget class="QMenu" name="menu_S">
    <property name="title">
     <string>设置(&amp;S)</string>
    </property>
    <addaction name="action"/>
   </widget>
   <widget class="QMenu" name="menu_H">
    <property name="title">
     <string>帮助(&amp;H)</string>
    </property>
    <addaction name="actionHelp"/>
   </widget>
   <widget class="QMenu" name="menu_M">
    <property name="title">
     <string>运动监测(&amp;M)</string>
    </property>
    <addaction name="action_3"/>
   </widget>
   <addaction name="menuYonghu"/>
   <addaction name="menu_M"/>
   <addaction name="menu_I"/>
   <addaction name="menu_S"/>
   <addaction name="menu_H"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionLogin">
   <property name="text">
    <string>登录</string>
   </property>
  </action>
  <action name="actionLogout">
   <property name="text">
    <string>注销</string>
   </property>
  </action>
  <action name="actionD">
   <property name="text">
    <string>奶山羊信息管理</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>获取帮助</string>
   </property>
  </action>
  <action name="action_Device_Query">
   <property name="text">
    <string>设备信息管理</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>备份设置</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>运动监测界面</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>绑定设置</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
